usl_pipeline/cloud_functions/main.py

```diff
@@ -289,15 +291,51 @@ def _build_feature_matrix_from_archive(
                 continue
 
             name = pathlib.PurePosixPath(member.name).name
+            # TODO: Group logic branch by path (per hazard model)
             if name == file_names.ELEVATION_TIF:
                 return _read_elevation_features(fd)
+            # Handle heat model input files (WPS)
+            elif name.startswith("met_em") and name.endswith(".nc"):
+                return _read_wps_features(fd)
             # TODO: handle additional archive members.
             else:
                 logging.warning(f"Unexpected member name: {name}")
 
     return None, FeatureMetadata()
 
 
+def _read_wps_features(fd: IO[bytes]) -> Tuple[NDArray, FeatureMetadata]:
+    # Ignore type checker error - BytesIO inherits from expected type BufferedIOBase
+    # https://shorturl.at/lk4om
+    with xr.open_dataset(fd) as ds:  # type: ignore
+        feature = ds.data_vars["SNOALB"].values
+        # Drop time axis - each WPS output file will correspond to single time value
+        feature = numpy.squeeze(feature, axis=0)
+        feature = _reorder_wps_spatial_axes(feature)
+        # TODO: Add lat/lon spatial coordinates to array
+
+    # TODO: Write to metastore
+    return feature, FeatureMetadata()
+
+
+def _reorder_wps_spatial_axes(feature: NDArray) -> NDArray:
+    """Reorders the axes for x-lat and y-lon.
```

Comment:
Can you describe this a bit more? What are the original axes and how do they need to be re-ordered? (and maybe what is AtmoML and x-lat & y-long and what's their relevance?)
It looks like we're taking a raster of shape (width, height) and returning one of (height, width) -- or maybe the other way around?

Reply:
Removing this for now - working on refining this in a separate branch